.orchestra/config/lib/cmake.lib.yml

```diff
@@ -11,6 +11,7 @@
   cmake \
     -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
     -DCMAKE_INSTALL_SO_NO_EXE=0 \
+    -DCMAKE_PREFIX_PATH="$ORCHESTRA_ROOT" \
```

Comment:
Placing `-DCMAKE_PREFIX_PATH="$ORCHESTRA_ROOT"` here instead of in `llvm_common.lib.yml` kinda scares me, did you try placing it there @pfez?